go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -295,32 +294,10 @@ func (c *Controller) syncServices(key string) error {
 			// mark the vip as processed
 			vipsTracked = vipsTracked.Delete(virtualIPKey(vip, svcPort.Protocol))
 
-			// handle reject ACLs for services without endpoints
-			// eventually we have to remove this because it will
-			// be implemented by OVN
-			// https://github.com/ovn-org/ovn-kubernetes/pull/1887
-			rejectACLName := loadbalancer.GenerateACLNameForOVNCommand(lbID, ip, svcPort.Port)
-			aclID, err := acl.GetACLByName(rejectACLName)
-			if err != nil {
-				klog.Errorf("Error trying to get ACL for Service %s/%s: %v", name, namespace, err)
-			}
-			// if there is no ACL and there is no endpoints we add a new ACL
-			// if there is an ACL and we have endpoints we have to remove the ACL
-			// if there is no ACL and we have endpoints we don´t need to do anything
-			if len(eps) == 0 && len(aclID) == 0 {
+			// reject ACLs are handled by OVN
+			if len(eps) == 0 {
 				klog.V(4).Infof("Service %s/%s without endpoints", name, namespace)
-				_, err = acl.AddRejectACLToPortGroup(c.clusterPortGroupUUID, rejectACLName, ip, int(svcPort.Port), svcPort.Protocol)
-				if err != nil {
-					klog.Errorf("Error trying to add ACL for Service %s/%s: %v", name, namespace, err)
-				}
-			} else if len(aclID) > 0 {
-				// remove acl
-				err = acl.RemoveACLFromPortGroup(aclID, c.clusterPortGroupUUID)
-				if err != nil {
-					klog.Errorf("Error trying to remove ACL for Service %s/%s: %v", name, namespace, err)
-				}
 			}
-			// end of reject ACL code
```

Comment:
The comment on `Controller.clusterPortGroupUUID` says

	// clusterPortGroupUUID contains the UUID of the port groups used for the services rejects ACLs
	clusterPortGroupUUID string

and there's still a bunch of stuff being done with that... is that irrelevant now?

Reply:
yeah, it should be rejected by OVN directly